lib/pyenv.rb

```diff
@@ -23,11 +23,13 @@ def setup!
       # To avoid starting multiple build jobs, acquire lock during installation
       synchronize("#{pyenv_root}.lock") do
         versions = capture(pyenv("versions", "--bare")).strip.split
-        unless versions.include?(version)
-          update!
-          listener << "Installing #{version}..."
-          run(pyenv("install", version), {out: listener})
-          listener << "Installed #{version}."
+        version.split(":").each do |version|
+          unless versions.include?(version)
+            update!
```

Comment:
Oops, you're completely right, I just saw this in the Jenkins job log too. Amending